src/modules/rlm_python/rlm_python.c

```diff
@@ -230,13 +230,13 @@ static PyObject *mod_log(UNUSED PyObject *module, PyObject *args)
 	char *msg;
 
 	if (!PyArg_ParseTuple(args, "is", &status, &msg)) {
-		return NULL;
+		Py_RETURN_NONE;
 	}
 
 	fr_log(&default_log, status, __FILE__, __LINE__, "%s", msg);
 	Py_INCREF(Py_None);
```

Comment:
Need to remove this too, else we leak nones.

Reply:
done